### ISSUE TITLE
Inputs and labels not shown inside collapsible

### ISSUE THREAD

**Issue by mef** [papercss/papercss#271]
Hi, there appears to be a bug in collapsible

Whenever an `input` or `label` is included inside the collapsible body, it does not appear on screen.

### Example code

```
<div class="collapsible">
    <input id="collapsible1" type="checkbox" name="collapsible">
    <label for="collapsible1">Example</label>
    <div class="collapsible-body">
      <span>
        Bacon ipsum text input
      </span>
      <input type="text" name="whatever" id="whatever" value="do you see me?">
    </div>
  </div>
```

### Expected result

![expected-situation](https://user-images.githubusercontent.com/925443/159673395-99140382-1872-49dd-b572-47ee8217b3dd.png)

### Observed result

![observed-situation](https://user-images.githubusercontent.com/925443/159673417-44cd54b0-26d8-4956-8efd-cc9cfe7405c9.png)

### Fix

Replacing selector `.collapsible input` by `.collapsible > input` for the `display:none` rule fixes the problem (similar correction to be applied for labels).

Would you like me to send a pull request with this correction?

**Comment by koester** [comment]
👍

**Comment by rhyneav** [comment]
Fixed in #277!
